src/simtools/applications/derive_limits.py

```diff
@@ -0,0 +1,99 @@
+#!/usr/bin/python3
+
+r"""
+Derives the limits for energy, radial distance, and viewcone using the LimitCalculator.
```

Comment:
I suggest to modify to

- "Derives limits for ... to be used in CORSIKA simulations"

Otherwise unclear for what it is used.